### ISSUE TITLE
"dialect" and "encoding" should be removed from block "resource"

### ISSUE THREAD

**Issue by wingechr** [OpenEnergyPlatform/oemetadata#42]
these are only relevant for uploading/parsing csv fiels and do not apply to data in the database

**Comment by jh-RLI** [comment]
OEMetadata also support [datapackes](https://specs.frictionlessdata.io/data-package/#language). Here the data can be available in csv format. We recommend .csv files in the new [OEdataModel-datapackage](https://github.com/OpenEnergyPlatform/oedatamodel/tree/develop/oedatamodel/latest/v110/datapackage/OEDataModel-concrete-datapackage). 
The use case often is that the data is provided from users in the format of .csv(excel) files, so the metadata should also support them. Furthermore, this information can be used in a download process to download the data package in the original format.

**Comment by wingechr** [comment]
I totally agree, but this is only to applicable to the exported data, not the data in the database. 

Example: I upload data from a csv file in iso-8859-1 encoding and with ";" as delimiters. This information has no meaning for the data in the database anymore. Furthermore, if you export it as a datapackage, it will be almost certainly encoded in utf-8. 

So this meta-data is not descibing data in the database but exported data (in csv/datapackage)

**Comment by jh-RLI** [comment]
I agree, we should not attach the meta-info to data in the database. Before the upload and after the download this information is relevant IMO.
Maybe we just strip the info from the JSON -> in the oep-client and oem2orm when uploading?  
Or we create a new version and add an explicit data package (metadata) folder/file and remove "dialect" from the current oemetadata.json. In this case, a new release would be necessary. @christian-rli

**Comment by christian-rli** [comment]
Hmm. I'm not sure I fully understand your second suggestion @jh-RLI . Stripping `dialect` and `encoding` when uploading seems very reasonable. The information is not relevant in the database, or even factually incorrect, as it describes the structure of the csv the data was taken from not the data in the database. The information can then be added upon export, which should be easy, as the export can always serve the same format. So I fully agree with you @wingechr . In my opinion the info can just be removed/added during up-/download. Can you work with that @jh-RLI

**Comment by jh-RLI** [comment]
I can work with it and implement it in oem2orm @christian-rli . I wanted to bring up an alternative suggestion, but it would go too far for this problem. I meant that in addition to the oemetadata there would be an oedatapackage version, so you would have one version for "database" data and one for exported data.
